src/boot_O2_g3/fault_drawer.c

```diff
@@ -113,37 +149,39 @@ void FaultDrawer_DrawChar(char c) {
 s32 FaultDrawer_ColorToPrintColor(u16 color) {
     s32 i;
 
-    for (i = 0; i < 10; i++) {
-        if (color == sFaultDrawContext->printColors[i]) {
+    for (i = 0; i < ARRAY_COUNT(sFaultDrawerInstance->printColors); i++) {
+        if (color == sFaultDrawerInstance->printColors[i]) {
             return i;
         }
     }
     return -1;
 }
 
-void FaultDrawer_UpdatePrintColor() {
+void FaultDrawer_UpdatePrintColor(void) {
     s32 idx;
 
-    if (sFaultDrawContext->osSyncPrintfEnabled) {
+    if (sFaultDrawerInstance->osSyncPrintfEnabled) {
         osSyncPrintf(VT_RST);
-        idx = FaultDrawer_ColorToPrintColor(sFaultDrawContext->foreColor);
+
+        idx = FaultDrawer_ColorToPrintColor(sFaultDrawerInstance->foreColor);
         if (idx >= 0 && idx < 8) {
```

Comment:
```suggestion
        if ((index >= 0) && (index < 8)) {
```
Just a small cleanup